Platforms/QemuQ35Pkg/Docs/Features/feature_prm.md

```diff
@@ -0,0 +1,33 @@
+# Platform Runtime Mechanism (PRM)
+
+## Overview
+
+Platform Runtime Mechanism (PRM) introduces the capability of moving certain classes of SMM code out of SMM and into
+a code module that executes within OS context. Generally, SMM code that does not depend upon SMM execution privileges
+is a candidate for conversion to PRM. The PRM conversion process involves porting code from SMM modules into code
+modules that execute at CPL0 and are directly invoked by host OS kernel components. These code modules are called
+**PRM Modules**. Functions within a PRM Module exposed to the OS for runtime execution are called **PRM Handlers**.
+
+End-to-end PRM support on a system requires firmware and OS support. The firmware must provide an initial set of PRM
+modules for the OS to use along with an ACPI tabled called PRMT that describes those modules. The OS uses this
+information to invoke PRM functionality when requested by a kernel component. PRM has two high-level invocation
+paths either directly from an OS driver (direct call) or by interacting with an ACPI OpRegion (ACPI call).

```

Comment:
Does it make any sense to list the modules and libraries that relate to this feature?

Reply:
Sure. I will add that.

Reply:
Updated in commit https://github.com/microsoft/mu_tiano_platforms/pull/5/commits/39229a0a04afc28c68d9f78a9b71af6994326c75